aiida/tools/importexport/common/archive.py

```diff
@@ -147,8 +147,8 @@ def get_info(self):
         :return: a dictionary with basic details
         """
         info = {
-            'version aiida': self.version_aiida,
-            'version format': self.version_format,
+            'aiida version': self.version_aiida,
+            'format version': self.version_format,
```

Comment:
attributes are also called `version_aiida` so why not keep this the same?

Reply:
It's just not the colloquial use (you would say "AiiDA version") but if that mirrors the attributes, it's fine to keep.